### ISSUE TITLE
Time Pieces

### ISSUE THREAD

**Issue by D0m0suka** [Bouncyrock/TaleSpire-Beta-AssetRequests#213]
Any other time period assets or props. I play Savage Worlds Adventure Addition, and the simple ruleset allows me to change themes rather easy, incorporating various other games pretty much on the fly. This, of course, lead to the fantastic opportunity of time travel in my RPG. I would love Ships, Cars, Motorcycles, Ancient Greece, 1930's America, Wild West and other time period assets. This would also help games that are not D&D oriented such as Rifts (Future/post apocalyptical), Cyberpunk (Future), Shadowrun (similar to Cyberpunk, but with magic), Call of Cthulu (Noir/1930's), Savage Worlds, Deadlands (Wild West), 50 Fathoms (Pirates), Warhammer (Similar "dead boys" and "Space Marines"), etc.
![SteampunkAirship](https://user-images.githubusercontent.com/83093071/116353065-c6be1100-a7bb-11eb-8f2b-2f1646d9b303.jpg)
Steam Punk era
![worlds-most-famous-shipwreck-1050x746](https://user-images.githubusercontent.com/83093071/116353078-c9b90180-a7bb-11eb-85a1-9efb5f3d3859.jpg)
50 Fathoms
![1930's RPG car token](https://user-images.githubusercontent.com/83093071/116353089-cd4c8880-a7bb-11eb-8d5e-756c01eb9fdb.jpg)
Call of Cthulu
![Cyberpunk-Cars-map-assets-Base-cars](https://user-images.githubusercontent.com/83093071/116353100-d2113c80-a7bb-11eb-835c-72c3ced654ef.jpg)
Cyberpunk/Rifts/Shadowrun
![unnamed](https://user-images.githubusercontent.com/83093071/116353130-e05f5880-a7bb-11eb-80cf-7a672896d052.jpg)
Cyberpunk/Rifts/Shadowrun/Futuristic Motorcycle
![20100501-crazy](https://user-images.githubusercontent.com/83093071/116353949-4ef0e600-a7bd-11eb-94a4-bc52949f1a22.jpg)
Rifts Crazy
![Juicer_Mega_large](https://user-images.githubusercontent.com/83093071/116353956-52846d00-a7bd-11eb-8aeb-bb76c6262b38.jpg)
Rifts Megajuicer

All of these things would be amazing, and are just some ideas. I know I'm kind of all over the place with the ideas, but as it is a time travel campaign, it is the nature of the beast. Thank you for your time and consideration.
If you support my Period Pieces idea/obsession, Thumbs Up, please!

**Comment by hadr13n** [comment]
Yes ! Anciant greece, or antiquity, or Mythology content packs would be i think quite original and useful since it is quite rare to come about in the ttrpg universe, and it would give Talespire an advantage, since they would be the "only ones" to propose this service
